packages/nameguard-sdk/src/index.ts

```diff
@@ -372,27 +376,50 @@ export interface ConsolidatedNameGuardReport extends ConsolidatedReport {
 /**
  * NameGuard report that contains the full results of all `checks` on all `labels` in a name.
  */
-export interface NameGuardReport extends ConsolidatedNameGuardReport {
+export interface AbstractNameGuardReport extends ConsolidatedNameGuardReport {
+  /* Whether or not the name was inspected. */
+  inspected: boolean;
+
   /* The results of all checks performed by NameGuard on `name`. */
   checks: CheckResult[];
 
-  /** Details of the inspection of all labels in `name`. */
-  labels: LabelGuardReport[];
+  /** Details of the inspection of all labels in `name`.
+    *
+    * `undefined` if `name` was uninspected and this is an `UninspectedNameGuardReport`.
+  */
+  labels?: LabelGuardReport[];
```

Comment:
@notrab Hey, this field now might be undefined. Can you please investigate and confirm if we need to update anything in `nameguard-react`, `examples.nameguard.io` or `nameguard.io`?